.gitignore

```diff
@@ -27,3 +27,6 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+
+# IntelliJ
+.idea
```

Comment:
We can keep this but wanted to let you know about [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) if you didn't know already, it's a nice way to ignore your editor files without having to do so in each new repository. Here's how it's setup on my config to ignore vim files:
```bash
# ~/.gitignore                                                                                                                   
*.sw?
.ctags
tags
```
and enable it your git config file, or you can also run `git config --global core.excludesfile ~/.gitignore`
```
# ~/.gitconfig
[core]
  excludesfile = /Users/{your_username}/.gitignore
```